colony/view/configure_list_view.py

```diff
@@ -0,0 +1,26 @@
+import tabulate
+
+from colony.constants import ColonyConfigKeys
+from colony.view.view_helper import mask_token
+
+
+class ConfigureListView:
+    def __init__(self, config: dict):
+        self.config = config
+
+    def render(self):
+        if not self.config:
+            return "Config file is empty. Use 'colony configure set' to configure Colony CLI."
+
+        result_table = []
+        for profile in self.config.keys():
+            result_table.append(
+                {
```

Comment:
I think we have to use OrderedDict here instead of regular dict.
Stable columns order in this case seems important to me. BTW, not sure if corresponding tests are always green here

Reply:
done